azurerm/internal/services/cdn/tests/cdn_endpoint_resource_test.go

```diff
@@ -82,10 +82,10 @@ func TestAccAzureRMCdnEndpoint_updateHostHeader(t *testing.T) {
 		CheckDestroy: testCheckAzureRMCdnEndpointDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccAzureRMCdnEndpoint_hostHeader(data, "www.example.com"),
+				Config: testAccAzureRMCdnEndpoint_hostHeader(data, "www.google.com"),
```

Comment:
[www.example.com](http://www.example.com) is explicitly meant to be used for demonstration purposes - is the API explicitly disallowing this? if so this feels like a bug in the API?

Reply:
After tested, seems I can create it successfully with same host name in azure portal. After checked api version, I found they are different. Azure portal is using 2020-04-15 but terraform is using 2019-04-15. I assume there is a bug in version 2019-04-15. So I filed an issue on Azure/azure-rest-api-specs#10090